_config.yml

```diff
@@ -215,37 +215,37 @@ get_started:
     children:
       - name: Why Choose Search.gov?
         url: "/about/why-choose-searchgov.html"
-      - name: Available Search Features
+      - name: Results Page Feature Comparison
         url: "/admin-center/display/hosted-vs-api-results.html"
     
 
 indexing:
-  - name: Index with Search.gov
+  - name: Index With Search.gov
     url: "/indexing/indexing-with-searchgov.html"
     children:
-      - name: How does indexing work?
+      - name: How Does Indexing Work?
         children:
-          - name: How search engines index your website (Webinar)
+          - name: How Search Engines Index Your Website (Webinar)
             url: "/get-started/training.html#how-engines-index"
-          - name: What Search.gov indexes from your website
+          - name: What Search.gov Indexes From Your Website
             url: "/indexing/what-searchgov-indexes.html"
-          - name: How Search.gov ranks your search results
+          - name: How Search.gov Ranks Your Search Results
             url: "/indexing/ranking-factors.html"
-          - name: How A Page on a Sitemap Becomes a Search
+          - name: How a Page on a Sitemap Becomes a Search Result
             url: "/indexing/sitemaps-to-search-sites.html"
-          - name: A step-by-step indexing workflow
+          - name: A Step-by-step Indexing Workflow
             url: "/indexing/indexing-workflow.html"
       - name: Domain Level SEO Supports
         children:
-          - name: All about XML sitemaps
+          - name: All About XML Sitemaps
             url: "/indexing/sitemaps.html"
-          - name: Robots.txt files
+          - name: \/robots.txt Files
```

Comment:
This displays with both the forward and backslash on the Federalist preview. Does it error if you don't try to escape it with the backslash?